spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2765,6 +2765,26 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("randn expression with random parameters") {
+    val partitionsNumber = Random.nextInt(10) + 1
```

Comment:
Why random values if we only run it once? I see that `"rand expression with random parameters"` already did this, but it feels like we could get lucky with a simple test case only running it one time. I guess I should double check if we seed this RNG (I know we modified some other tests to use fixed seeds).

Reply:
Are there specific configs that seem more important to test than others? I'd want to make sure we hit those. If it's a bit more arbitrary, then I think the current setup is fine.